### PR TITLE
feat: Update link styles

### PR DESCRIPTION
**Pull request by adamledwards**
Update link styles for hover states and active and `:active` state to be black

### CODE REVIEW

File: e2e/tests/__screenshots__/visual-regression.spec.ts/get-started/design-library/js-disabled--firefox.png
Comment (adamledwards) [inline e2e/tests/__screenshots__/visual-regression.spec.ts/get-started/design-library/js-disabled--firefox.png:1]: Changes made to the side-nav current state

File: e2e/tests/__screenshots__/visual-regression.spec.ts/get-started/change-the-image-assets-path/js-disabled--firefox.png
Comment (adamledwards) [inline e2e/tests/__screenshots__/visual-regression.spec.ts/get-started/change-the-image-assets-path/js-disabled--firefox.png:1]: Changes made to the primary nav and secondary nav current state

**Review by jpveooys — COMMENTED** [review]
Nice work! Think there were some updates needed for the hero button as well (hover colour at least)

**Review by jpveooys — APPROVED** [review]
:+1:

**Pull request merged**